tests/roaring64_unit.cpp

```diff
@@ -952,6 +952,27 @@ DEFINE_TEST(test_andnot_inplace) {
     }
 }
 
+DEFINE_TEST(test_shrink_grow_48_art_node) {
```

Comment:
Since this is purely testing ART, what about moving this test into `art_unit.cpp`?